### PR TITLE
Relay Initial 

### PR DESCRIPTION
**Pull request by enginaryum**

### CODE REVIEW

File: src/__generated__/AppRendererQuery.graphql.js [hunk 1]
```diff
@@ -0,0 +1,337 @@
+/**
+ * @flow
+ * @relayHash 8e5135ef99f3ad4b86776d4ee4b2bddc
+ */
+
+/* eslint-disable */
+
```
[the remark this answers is not in the document]
Comment (EddyIonescu) [reply src/__generated__/AppRendererQuery.graphql.js]: If generated files are needed, you can modify `start.js` to generate them for us.

**Comment by EddyIonescu** [comment]
So the linting commit I made renames some files, so you'll have to move stuff around.
Also, I've made this PR (https://github.com/trynmaps/tryn-react/pull/17), which shows how we should be implementing this - that's how future additions to the middleware should be made too :)

File: package.json [hunk 2]
```diff
@@ -26,6 +26,7 @@
     "extract-text-webpack-plugin": "3.0.0",
     "file-loader": "0.11.2",
     "fs-extra": "3.0.1",
+    "get-graphql-schema": "^2.1.1",
```
[the remark this answers is not in the document]
Comment (enginaryum) [reply package.json]: actually this package clones graphql schema on given url. 
i was planning to use this on npm script as following to keep track of changes on api: 

'relay': "get-graphql-schema  http://35.192.24.65/graphql?  > ./schema.graphql && relay-compiler --src ./src --schema ./schema.graphql"

but i think we won't make much of change on schema, so we can just remove it out.

File: schema.graphql [hunk 3]
```diff
@@ -0,0 +1,66 @@
+type NextVehicle {
+    timeToArrival: Float
+    vehicle: Vehicle
+}
+
```
[the remark this answers is not in the document]
Comment (enginaryum) [reply schema.graphql]: to create __generated files, relay-compiler needs a schema

File: src/App.js [hunk 4]
```diff
@@ -41,20 +41,26 @@ class App extends Component {
...
     console.log(muniRoutesGeoJson);
-    const { viewport, settings } = this.state;
+    const {viewport, settings} = this.state;
     return (
-      <ReactMapGL {...viewport} 
-        mapStyle="mapbox://styles/eddyionescu/cj9btlvm2423a2sprnbgpk9r5"
-        mapboxApiAccessToken={MAPBOX_ACCESS_TOKEN}
-        onViewportChange={this._onViewportChange} 
+      <ReactMapGL {...viewport}
+                  mapStyle="mapbox://styles/eddyionescu/cj9btlvm2423a2sprnbgpk9r5"
```
[the remark this answers is not in the document]
Comment (enginaryum) [reply src/App.js]: if you guys using JetBrains IDE, can you share your settings please?

File: package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (EddyIonescu) [reply package.json]: So I'd rather directly use the schema from tryn-api. That's what I did in my PR :)

File: schema.graphql (hunk 3 above)
[the remark this answers is not in the document]
Comment (EddyIonescu) [reply schema.graphql]: Same as above, just use it directly from tryn-api.

File: src/__generated__/AppRendererQuery.graphql.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (EddyIonescu) [reply src/__generated__/AppRendererQuery.graphql.js]: Just do `yarn run relay --watch` when working on tryn-react.

File: src/App.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (EddyIonescu) [reply src/App.js]: We now have linting based on the Airbnb style guide (enforced by eslint) see master for details.

**Pull request closed without merging**